mitodl/mitxpro-zendesk-theme
# Sync Xpro Zendesk theme with the upstream.

### What are the relevant tickets?

[#3842](https://github.com/mitodl/hq/issues/3842) 


### Description (What does it do?)
Here is the Testing URL of re-synced theme: https://mit7400.zendesk.com/hc/en-us

This PR syncs the Xpro Zendesk theme with the upstream. It maintains the existing design sync and incorporates updates and features from the Zendesk upstream theme. To see a list of all the updates and features, please refer to the changelog [here](https://github.com/zendesk/copenhagen_theme/blob/master/CHANGELOG.md). 

### How should this be manually tested?

1. Homepage Changes:

- Visit the [old xpro Zendesk home page](https://xpro.zendesk.com/hc/en-us) and compare it with the [new xpro Zendesk home page](https://mit7400.zendesk.com/hc/en-us).
- Ensure that the clear search feature is functional on multiple pages.
- Confirm that the search submit button on the homepage is enabled and working.

2. Submit a Request Page:

- Navigate to the [old submit a request page](https://xpro.zendesk.com/hc/en-us/requests/new) and compare it with the [new submit a request page](https://mit7400.zendesk.com/hc/en-us/requests/new).

- Ensure that the layout and functionality of the new submit a request page match the expected changes and incorporate new changes if any.

3. "Sign In" link:
- Confirm that the "Sign In" link is not visible in the test environment and will not be present when deployed to production or verify that the "Sign In" link is disabled in the header for both desktop and mobile views. 

Overall Testing:

- Conduct general testing of the Zendesk theme to ensure that all existing features are functioning correctly after the sync with the upstream.
- Test any other specific features or functionalities mentioned in the changelog or observed during the sync process.

### Screenshots (if appropriate):

**Old xpro zendesk home page:**
![image](https://github.com/mitodl/mitxpro-zendesk-theme/assets/41048311/a8af2054-4c04-4e80-a7f9-6360f3e38050)

**New xpro zendesk home page:**
![image](https://github.com/mitodl/mitxpro-zendesk-theme/assets/41048311/4837b03d-d7e6-4e91-b2f0-22ae7a7968fc)

**Old submit a request page:**
![image](https://github.com/mitodl/mitxpro-zendesk-theme/assets/41048311/915a2384-4be4-4afe-aebb-db24095bf107)

**New submit a request page:**
![image](https://github.com/mitodl/mitxpro-zendesk-theme/assets/41048311/39ccac06-d2bf-415f-b505-9fd4ddab7a1f)

**Old xpro zendesk categories page:**
![image](https://github.com/mitodl/mitxpro-zendesk-theme/assets/41048311/181ddfbf-123e-48e3-a3c4-cef3cb132ed1)


**New xpro zendesk categories page:**
![image](https://github.com/mitodl/mitxpro-zendesk-theme/assets/41048311/2d3bc866-1e8a-42eb-87bf-405a8a1c167d)

**Old mobile view:**
![image](https://github.com/mitodl/mitxpro-zendesk-theme/assets/41048311/640f19b7-f07b-478a-badf-a41b9b3539b7)


**New mobile view:**
![image](https://github.com/mitodl/mitxpro-zendesk-theme/assets/41048311/3b9e04ae-7993-47c0-be59-5962594a3934)


### Additional Context

- It seems the updated logo PR [#13](https://github.com/mitodl/mitxpro-zendesk-theme/pull/13) has not been deployed to the live instance, so there will be discrepancies in the logos when comparing screenshots provided above.
- I've also found some branding inconsistencies as shown in the screenshots below. These inconsistencies exist on other pages as well. So fixes for them are included in this same PR.

<img width="1250" alt="image" src="https://github.com/mitodl/mitxpro-zendesk-theme/assets/41048311/3f52d2fa-6922-4aa8-a7b0-64ccc3652c01">

<img width="1250" alt="Screenshot 2024-06-03 at 11 05 09 PM" src="https://github.com/mitodl/mitxpro-zendesk-theme/assets/41048311/ceaa49c6-49d9-4ccb-bfed-1d9c115a0061">
